Jenkinsfile

```diff
@@ -14,13 +14,53 @@
  * limitations under the License.
  */
 
+
+//node-affinity
+//nodes 1..3 are reserved for Jenkins slave pods.
+//node 0 is used for the Jenkins master
+//node 4 is used for SQ and Nexus
```

Comment:
I don't think we need to know that node 4 is used for SQ and Nexus. But knowing about Jenkins slaves and master are useful
```suggestion
```